new/src/sections/portfolio/portfolio.tsx

```diff
@@ -1,7 +1,79 @@
-const Portfolio = (): JSX.Element => {
+import HeaderIcon from "../../assets/icons/portfolio-icon.png";
+import PortfolioItem from "./portfolioItem";
+import Thumbnail2021 from "../../assets/portfolio-thumbnails/A2wJky66rH_cover.png";
+import Thumbnail2020 from "../../assets/portfolio-thumbnails/kitten_mitten_cover.png";
+import Thumbnail2019 from "../../assets/portfolio-thumbnails/memory_lanes_cover.png";
+import Thumbnail2018 from "../../assets/portfolio-thumbnails/cogworks_cover.png";
+const Webm2021 = require("../../assets/portfolio-thumbnails/A2wJky66rH.webm");
+const Webm2020 = require("../../assets/portfolio-thumbnails/kittenmitten.webm");
+const Webm2019 = require("../../assets/portfolio-thumbnails/memorylanes.webm");
+const Webm2018 = require("../../assets/portfolio-thumbnails/cogworks.webm");
+
+const portfolioItems = [
```

Comment:
Please also specify the type of each item, so that others can ensure each item has specific properties.

```ts
type PortfolioItemType = {
  link: string,
  videoSrc: string,
  // Other properties
};

const porfolioItems: PortfolioItemType[] = [
  // Your items
];
```